lib/charms/loki_k8s/v0/charm_logging.py

```diff
@@ -0,0 +1,466 @@
+#!/usr/bin/env python3
+# Copyright 2022 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+"""This charm library contains utilities to automatically forward your charm logs to a loki-push-api endpoint.
+
+(yes! charm code, not workload code!)
+
+This means that, if your charm is related to, for example, COS' Loki charm (or a Grafana Agent),
+you will be able to inspect in real time from the Grafana dashboard the logs emitted by your charm.
+
+To start using this library, you need to do two things:
```

Comment:
I think that the dependence on `loki_push_api` could be more emphasized. Maybe something like:

1) if your charm isn't already related to Loki using any of the consumers/forwarders from `loki_push_api` library, you need to fetch the library:

`charmcraft fetch-lib charms.loki_k8s.v1.loki_push_api`

and add the logging consumer that matches your use case. See https://charmhub.io/loki-k8s/libraries/loki_push_apihttps://charmhub.io/loki-k8s/libraries/loki_push_api for more information.